cpu/lpc2387/Kconfig

```diff
@@ -0,0 +1,44 @@
+# Copyright (c) 2020 HAW Hamburg
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+#
+
+config CPU_FAM_LPC23XX
+    bool
+    select CPU_CORE_ARM7TDMI_S
+    select HAS_BACKUP_RAM
+    select HAS_CPU_LPC2387
+    select HAS_PERIPH_DAC
+    select HAS_PERIPH_GPIO
+    select HAS_PERIPH_GPIO_IRQ
+    select HAS_PERIPH_TIMER_PERIODIC
+
+## CPU Models
+config CPU_MODEL_LPC2387
+    bool
+    select CPU_FAM_LPC23XX
+
+config CPU_MODEL_LPC2388
+    bool
+    select CPU_FAM_LPC23XX
+
+## Declaration of specific features
+config HAS_CPU_LPC2387
+    bool
+    help
+        Indicates that an 'lpc2387' cpu is being used.
+
+## Common CPU symbols
+config CPU_FAM
+    default "lpc23xx" if CPU_FAM_LPC23XX
+
+config CPU_MODEL
+    default "lpc2387" if CPU_MODEL_LPC2387
+    default "lpc2388" if CPU_MODEL_LPC2388
+
+config CPU
+    default "lpc2387" if CPU_FAM_LPC23XX
```

Comment:
Maybe we should rename the directory and Doxygen group to `lpc23xx`. Then we could have `CPU = lpc23xx` and CPU_MODEL could be either `lpc2387` or `lpc2388`.